### ISSUE TITLE
Error when creating new user group

### ISSUE THREAD

**Issue by milesjbland** [ryssbowh/craft-activity#8]
Hi there,

I'm getting the following error when trying to create a new user group:

```
yii\base\UnknownMethodException: Calling unknown method: Ryssbowh\Activity\recorders\UserSettings::onChanged() in /vendor/yiisoft/yii2/base/Component.php:300
Stack trace:
#0 /vendor/ryssbowh/craft-activity/src/recorders/UserSettings.php(17): yii\base\Component->__call('onChanged', Array)
#1 /vendor/craftcms/cms/src/services/ProjectConfig.php(1199): Ryssbowh\Activity\recorders\UserSettings->Ryssbowh\Activity\recorders\{closure}(Object(craft\events\ConfigEvent))
#2 [internal function]: craft\services\ProjectConfig->handleChangeEvent(Object(craft\events\ConfigEvent))
#3 /vendor/yiisoft/yii2/base/Component.php(633): call_user_func(Array, Object(craft\events\ConfigEvent))
#4 /vendor/craftcms/cms/src/services/ProjectConfig.php(771): yii\base\Component->trigger('updateItem', Object(craft\events\ConfigEvent))
#5 /vendor/craftcms/cms/src/services/ProjectConfig.php(1189): craft\services\ProjectConfig->_processConfigChangesInternal('users')
#6 [internal function]: craft\services\ProjectConfig->handleChangeEvent(Object(craft\events\ConfigEvent))
#7 /vendor/yiisoft/yii2/base/Component.php(633): call_user_func(Array, Object(craft\events\ConfigEvent))
#8 /vendor/craftcms/cms/src/services/ProjectConfig.php(771): yii\base\Component->trigger('updateItem', Object(craft\events\ConfigEvent))
#9 /vendor/craftcms/cms/src/services/ProjectConfig.php(1189): craft\services\ProjectConfig->_processConfigChangesInternal('users.groups.66...')
#10 [internal function]: craft\services\ProjectConfig->handleChangeEvent(Object(craft\events\ConfigEvent))
#11 /vendor/yiisoft/yii2/base/Component.php(633): call_user_func(Array, Object(craft\events\ConfigEvent))
#12 /vendor/craftcms/cms/src/services/ProjectConfig.php(768): yii\base\Component->trigger('addItem', Object(craft\events\ConfigEvent))
#13 /vendor/craftcms/cms/src/services/ProjectConfig.php(544): craft\services\ProjectConfig->_processConfigChangesInternal('users.groups.66...', true, 'Update permissi...')
#14 /vendor/craftcms/cms/src/services/UserPermissions.php(320): craft\services\ProjectConfig->set('users.groups.66...', Array, 'Update permissi...')
#15 /vendor/craftcms/cms/src/controllers/UserSettingsController.php(92): craft\services\UserPermissions->saveGroupPermissions(3, Array)
#16 [internal function]: craft\controllers\UserSettingsController->actionSaveGroup()
#17 /vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#18 /vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#19 /vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('save-group', Array)
#20 /vendor/craftcms/cms/src/web/Application.php(293): yii\base\Module->runAction('user-settings/s...', Array)
#21 /vendor/craftcms/cms/src/web/Application.php(602): craft\web\Application->runAction('user-settings/s...', Array)
#22 /vendor/craftcms/cms/src/web/Application.php(272): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#23 /vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#24 /web/index.php(22): yii\base\Application->run()
#25 /.composer/vendor/laravel/valet/server.php(234): require('...')
#26 {main}
```

Doesn't seem to matter what permissions I have checked/unchecked.

Might be related to: #4 

Craft CMS: **3.7.61**
User Activity: **1.2.4**

Let me know if you need anymore info!

**Comment by ryssbowh** [comment]
Hi,

This is now fixed in 1.2.6

**Comment by milesjbland** [comment]
Hey @ryssbowh,

Just updated and it's resolved. Thank you!
